act/api/helpers.py

```diff
@@ -27,19 +27,28 @@ def as_list(value):
 
 
 @functools.lru_cache(4096)
-def handle_fact(fact: Fact, output_format="json") -> None:
+def handle_fact(fact: Fact, output_filehandle: TextIO = None, output_format="json") -> None:
```

Comment:
I suggest that you move output_filehandle as the last argument, in case someone are currently using handle_fact without named arguments

Reply:
Smart! For me, it was more important for them to be in alphabetical order :P